ivy/array/linear_algebra.py

```diff
@@ -75,6 +75,90 @@ def cross(
         *,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
+        """Computes the cross product of 3-element vectors. If the array calling the function
```

Comment:
you should mention that the docstring has been borrowed from the Array API standard, as explained in the docstring section.